File: doc/src/scripts/Connecting.cs
using System.Net;
using KRPC.Client;
using KRPC.Client.Services.KRPC;

class Program
{
    public static void Main ()
    {
        using (var connection = new Connection (
            name : "Example", address: IPAddress.Parse ("10.0.2.2"),
            rpcPort: 1000, streamPort: 1001)) {
            var krpc = connection.KRPC ();
            System.Console.WriteLine (krpc.GetStatus ().Version);
        }
    }
}

File: 0.5.2/_static/client/java/Connecting.java
import krpc.client.Connection;
import krpc.client.RPCException;
import krpc.client.services.KRPC;

import java.io.IOException;

public class Connecting {
  public static void main(String[] args) throws IOException, RPCException {
    try (Connection connection = Connection.newInstance("Remote example", "my.domain.name", 1000, 1001)) {
      System.out.println(KRPC.newInstance(connection).getStatus().getVersion());
    }
  }
}
